src/drive/mobile/containers/UploadProgression.jsx

```diff
@@ -1,13 +1,66 @@
+import React from 'react'
 import { connect } from 'react-redux'
-import UploadProgression from '../components/UploadProgression'
-
-const mapStateToProps = state => ({
-  message: state.mobile.mediaBackup.currentUpload
-    ? state.mobile.mediaBackup.currentUpload.message
-    : undefined,
-  messageData: state.mobile.mediaBackup.currentUpload
-    ? state.mobile.mediaBackup.currentUpload.messageData
-    : undefined
-})
-
-export default connect(mapStateToProps)(UploadProgression)
+import styles from '../styles/uploadprogression'
+import { translate } from 'cozy-ui/react/I18n'
+
+const percent = (actual, total) => Math.floor(actual * 100 / total)
+
+const Progress = ({ percent, style, color, background }) => {
+  const containerStyle = {
+    opacity: percent < 100 ? 1 : 0,
+    transition: `${0.4}s opacity}`,
+    transitionDelay: `${percent < 100 ? 0 : 0.4}s`,
+    height: `${2}px`,
+    backgroundColor: background ? 'lightgray' : ''
+  }
+  const barStyle = {
+    display: 'inline-block',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    width: `${percent}%`,
+    maxWidth: '100%',
+    height: `${2}px`,
+    borderRadius: '0 1px 1px 0',
+    transition: `${0.4}s width, ${0.4}s background-color`,
+    backgroundColor: color || 'darkblue',
+    ...style
+  }
+  return (
+    <div style={containerStyle}>
+      <div style={barStyle} />
+    </div>
```

Comment:
Shouldn't we use the `progress` element for the markup, and css for styling it ?

Reply:
TIL https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
Not sure it's very applicable as
- hard to style
- not very standard
(cf. https://css-tricks.com/html5-progress-element/)

Reply:
We use them as part of the onboarding for mesinfos/cozy-beta, and they seem to work aright :) They are styled too!